resources/analysisTools/snvPipeline/snvAnnotation.sh

```diff
@@ -194,22 +199,7 @@ mkfifo ${npConfidence}
 if [[ ${isNoControlWorkflow-false} == "false" ]]; then
     cat ${filenameSNVVCF} > ${npConfidence} &
 else
-    cat ${filenameSNVVCF} | ${PYPY_BINARY} -u ${TOOL_CONFIDENCE_ANNOTATION} ${noControlFlag} -i - ${CONFIDENCE_OPTS} -a 0 > ${npConfidence} &
-fi
-
-# Needed for CFFI
-export C_INCLUDE_PATH=${HTSLIB_INCLUDE_PATH}
-module switch htslib/${HTSLIB_VERSION_FOR_HTS_PYTHON}
```

Comment:
Have you tested the impact of removing the module switch?

Reply:
everything tested